config/locales/en.yml

```diff
@@ -1133,10 +1133,17 @@ en:
     logged_out: "Successfully logged out."
   notes_controller:
     research_notes: "Research notes"
-    thank_you_for_contribution: "Success! Thank you for contributing open research,
-      and thanks for your patience while your post is approved by <a href='/wiki/moderation'>community
-      moderators</a> and we'll email you when it is published. In the meantime, if
-      you have more to contribute, feel free to do so."
+  thank_you_for_contribution: |
+      <h2>Success! You did it! Hooray!</h2>
```

Comment:
hi @banjo-tech seems there is some syntax error, i guess its indentation - the `<` of h2 should be indented 2 spaces deep _inline with the `a` in "thank" above, could you please check.  Thanks for working on this

Reply:
Hi @cesswairimu I do not think I understand what the error is but I have h2 and anchor tag are on the same line now.

Reply:
I meant line 1137 maybe should start inline with the `a` of "thank" on line 1136 ... does that make sense?

Reply:
line 1136 doesn't have an anchor tag
are you suggesting I add an anchor tag to line 1136?

Reply:
I meant like so https://github.com/publiclab/plots2/pull/11532/commits/700b6395faa3d3ebdec1f3dc352938bb3f6d8513 . Lets see if the tests will pass, thanks